src/librustdoc/html/render/mod.rs

```diff
@@ -1746,8 +1746,7 @@ fn print_sidebar(cx: &Context<'_>, it: &clean::Item, buffer: &mut Buffer) {
         path = relpath
     );
     if parentlen == 0 {
-        // There is no sidebar-items.js beyond the crate root path
-        // FIXME maybe dynamic crate loading can be merged here
+        write!(buffer, "<script defer src=\"top-level-sidebar-items.js\"></script>",);
     } else {
         write!(buffer, "<script defer src=\"{path}sidebar-items.js\"></script>", path = relpath);
     }
```

Comment:
I think what we really want here, rather than generating a file with a different name, is to generate a file named `sidebar-items.js` in the doc root, and unconditionally write a script tag that loads it. So these two `if` branches would get collapsed into one.

Reply:
That's better yeah